### PR TITLE
Set up very basic Docsy website

### PR DESCRIPTION
**Pull request by dholbach**
Based on https://github.com/weaveworks/kured/#readme

To test locally, use the [extended build of Hugo](https://gohugo.io/getting-started/installing/) and run `hugo server` in the branch.

As soon as we can, we will get this hooked up to a Netlify instance through CNCF. 

Signed-off-by: Daniel Holbach <daniel@weave.works>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,21 @@
...
+
+## How to modify this website
+
+The main landing page of this website can be modified in `config.yaml`.
+
+All of the content lives in the `content/` directories.
+
+## Running the site locally
+
+In order to run the Flux site locally, you need to install the [Hugo](https://gohugo.io) static site generator. Make sure to [install](https://gohugo.io/getting-started/installing/) the "extended" variant of Hugo with support for the [Hugo Pipes](https://gohugo.io/hugo-pipes/introduction/) feature.
```
Comment (evrardjp) [inline README.md:13]: Flux is mentioned here.

**Review by evrardjp — CHANGES_REQUESTED** [review]
few nits in my comments.
In addition to that, maybe we should mention how to contribute  to the website,helm charts, and kured?

File: README.md (hunk 1 above)
Comment (dholbach) [reply README.md]: oops :-)

**Comment by dholbach** [comment]
> few nits in my comments. In addition to that, maybe we should mention how to contribute to the website,helm charts, and kured (the organisation's structure)?

Can we extend the docs in a follow-up PR?

**Comment by dholbach** [comment]
→ https://deploy-preview-1--kured.netlify.app/

File: content/docs/community.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+title: Community
+---
+
+## Getting Help
+
+If you have any questions about, feedback for or problems with `kured`:
+
+- Invite yourself to the <a href="https://slack.cncf.io/" target="_blank">CNCF Slack</a>.
+- Ask a question on the [#kured](https://cloud-native.slack.com/archives/kured) slack channel.
+- [File an issue](https://github.com/weaveworks/kured/issues/new).
```
Comment (ckotzbauer) [inline content/docs/community.md:11]: Wrong issue link

File: netlify.toml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+[build]
+  publish = "public"
+  command = "make production-build"
+
+[build.environment]
+  GO_VERSION = "1.18"
```
Comment (ckotzbauer) [inline netlify.toml:6]: Any reason this differs from the version in go.mod?

**Review by ckotzbauer — COMMENTED** [review]
Only two small comments.
The website looks pretty good, it's a good starting point we can improve and extend later.

**Review by ckotzbauer — APPROVED** [review]

**Pull request merged**